src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php

```diff
@@ -224,11 +224,11 @@ private function getMergeSql()
     {
         $platform = $this->con->getDatabasePlatform()->getName();
 
-        switch ($platform) {
-            case 'mysql':
+        switch (true) {
```

Comment:
bonus fix: https://github.com/symfony/symfony/pull/19048/files#diff-e6a5cc5d66fa241233b3314fcd49f228R236 is dead code today